test/utils.sh

```diff
@@ -154,3 +157,7 @@ parse_test_output() {
     exit 1
   fi
 }
+
+cleanup() {
+  echo "Line:$LINENO"
```

Comment:
`$BASH_COMMAND` would be this helpful as well ?

Reply:
> echo "Line:$LINENO"

Doesn't it miss context? Line what?

`Script errored at line ${LINENO}: ${BASH_COMMAND}`

Reply:
Thanks! Updated.